workflow-examples/src/main/java/com/redhat/parodos/examples/prebuilt/migrationtoolkit/MigrationAssessmentWorkflow.java

```diff
@@ -0,0 +1,106 @@
+package com.redhat.parodos.examples.prebuilt.migrationtoolkit;
+
+import java.net.URI;
+import java.util.List;
+
+import com.redhat.parodos.tasks.migrationtoolkit.CreateApplicationTask;
+import com.redhat.parodos.tasks.migrationtoolkit.GetAnalysisTask;
+import com.redhat.parodos.tasks.migrationtoolkit.GetApplicationTask;
+import com.redhat.parodos.tasks.migrationtoolkit.SubmitAnalysisTask;
+import com.redhat.parodos.workflow.annotation.Assessment;
+import com.redhat.parodos.workflow.annotation.Checker;
+import com.redhat.parodos.workflow.annotation.Parameter;
+import com.redhat.parodos.workflow.option.WorkFlowOption;
+import com.redhat.parodos.workflow.parameter.WorkParameterType;
+import com.redhat.parodos.workflow.task.infrastructure.Notifier;
+import com.redhat.parodos.workflows.workflow.WorkFlow;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+
+import static com.redhat.parodos.workflows.workflow.SequentialFlow.Builder.aNewSequentialFlow;
```

Comment:
no static imports should be allowed in the code. weird the checkstyle didn't fail for it.
it should be allowed only in tests (unless workflow-examples are excluded).

Reply:
why?

Reply:
https://github.com/spring-projects/spring-framework/wiki/Code-Style#import-statements

> Static imports should not be used in production code, but they should be used in test code, especially for things like import static org.assertj.core.api.Assertions.assertThat;.

workflow-example is considered to be production code